src/api/ingest/serializers.py

```diff
@@ -52,6 +43,10 @@ def assign_choices(self, row, choices_sets):
                 continue
             try:
                 val = self._lower(val)
+                choices = {
+                    label.lower().replace("\t", " "): value
```

Comment:
This was just refactored from what was in `utils.py` previously... I have a memory that we put it in because of some kind of error that happened, but agree it's ugly. I'll take it out and we can see if it happens again.